components/directional-light.js

```diff
@@ -10,6 +10,6 @@ export default (options) => ({
   bias: 0.1,
   castShadows: true,
   radius: 1,
-  // shadowMapSize: 2048,
+  // shadowMapSize: 2048, //TODO: MARCIN: why is this commented out?
```

Comment:
Because we have default value and it is commented here to show it exists when/if we want to test things.